index/index.go

```diff
@@ -781,9 +781,10 @@ func ReadOffsetTable(bs ByteSlice, off uint64, f func([]string, uint64) error) e
 	d := encoding.NewDecbufAt(bs, int(off), castagnoliTable)
 	cnt := d.Be32()
 
+	keys := make([]string, 0, 2)
```

Comment:
https://github.com/prometheus/tsdb/pull/627#discussion_r299344697 tells why I have set the size to 2.